changelogs/fragments/2177-java_keystore_1668_dont_expose_secrets_on_cmdline.yml

```
@@ -0,0 +1,4 @@
+---
+bugfixes:
```

I think it is ok to label this as what it is:
```suggestion
security_fixes:
```